drive:
  map:
    max_driving_effort: 1.0
    min_driving_effort: -1.0
    turning_p: 10.0
  odom:
    max_driving_effort: 1.0
    min_driving_effort: -1.0
    turning_p: 10.0

gate:
  stop_thresh: 0.2
  drive_fwd_thresh: 0.34
  approach_distance: 2.0

search:
  stop_thresh: 0.2
  drive_fwd_thresh: 0.34

single_fiducial:
  stop_thresh: 0.7
  fiducial_stop_threshold: 1.75

waypoint:
  stop_thresh: 0.5
  drive_fwd_thresh: 0.34
  no_fiducial: -1